website/content/docs/agent/telemetry.mdx

```diff
@@ -89,6 +89,17 @@ These are some metrics emitted that can help you understand the health of your c
 
 **What to look for:** For a healthy cluster, you're looking for a `lastContact` lower than 200ms, `leader` > 0 and `candidate` == 0. Deviations from this might indicate flapping leadership.
 
+### Certificate Authority Expiration
+
+| Metric Name                            | Description                                                             | Unit    | Type  |
+| :------------------------- | :---------------------------------------------------------------------------------- | :------ | :---- |
+| `consul.mesh.active-root-ca.expiry`    | The number of seconds until the root CA expires, updated every hour.    | seconds | gauge |
+| `consul.mesh.active-signing-ca.expiry` | The number of seconds until the signing CA expires, updated every hour. | seconds | gauge |
+
+** Why they're important:** An externally managed CA, like with Vault, means that the renewal is handled outside of Consul and it will need to be monitored for expiration (and rotated as appropriate). 
```

Comment:
```suggestion
** Why they're important:** When using an external system, such as Vault, to manage the CA, you should monitor renewals to ensure that certificates do not expire unexpectedly and so that you can rotate as necessary. 
```
Use "like" when comparing things and "such as" when you are providing an example. Use present tense and active per the style guide.